.gitignore

```diff
@@ -1,5 +1,5 @@
 # Temporary/hidden files
-.*.swp
+.*
```

Comment:
This is minor, because Git never ignores files that are actually committed (well, technically, those that are in the index), but...

If you mark `.*` like this it might be good to explicitly un-ignore the dot files we have, i.e.:

    !.codecov.yml
    !.gitattributes
    !.gitignore
    !.hooks/
    !.travis.yml

(In particular this means if someone runs `git rm -r .; git add .` someday, the wanted dot-files get re-added to the index.)

Reply:
I wouldn't worry about this, since git add .` will warn that files are not being ignored.